app/pulp/app/serializers/user.py

```diff
@@ -0,0 +1,29 @@
+from rest_framework import serializers
+
+from pulp.app import models
+from pulp.app.serializers import ModelSerializer
+
+
+class UserSerializer(ModelSerializer):
+    username = serializers.CharField(
+        help_text="Required. 150 characters or fewer. Letters, digits and @/./+/-/_ only."
+    )
+
+    is_superuser = serializers.BooleanField(
+        help_text="Designates that this user has all permissions without explicitly assigning "
+                  "them."
+    )
+
+    groups = serializers.ManyRelatedField(
+        help_text="The groups this user belongs to. A user will get all permissions granted to "
+                  "each of their groups."
+    )
```

Comment:
We should not be using ManyRelatedField:
https://github.com/tomchristie/django-rest-framework/blob/master/rest_framework/relations.py#L441-L452

`groups` and `user_permissions` should (probably) either be HyperlinkedRelatedFields, be nested, or not exist.